contrib/coredns/policy/policy_test.go

```diff
@@ -0,0 +1,30 @@
+package policy
+
+import (
+	"context"
+	pb "github.com/infobloxopen/policy-box/pdp-service"
+	"github.com/infobloxopen/policy-box/pep"
+	"github.com/miekg/dns"
+	"testing"
+)
+
+// This function unit tests handlePermit function
+// Inputs are t *testing.T
+// Output is update in t
+func TestHandlePermit(t *testing.T) {
+	p := new(PolicyMiddleware)
+	p.pdp = new(pep.Client)
```

Comment:
For now I have removed the Client, which is not required for this Unit test. For next Unit test, will create a Client and create a test Client